### PR TITLE
feat(erlang highlight): support erlang highlight

### PR DESCRIPTION
**Pull request by Decodetalkers**
support erlang highlight

### CODE REVIEW

File: queries/erlang/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,75 @@
+[
+  "catch"
```
Comment (theHamsta) [inline queries/erlang/highlights.scm:2]: `@exception`?

File: queries/erlang/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  "of"
+  "case"
+  "when"
+  "after"
+  "receive"
+  "end"
+  "module"
+  "export"
```
Comment (theHamsta) [inline queries/erlang/highlights.scm:10]: I don't erlang. maybe `module`, `export` fit well in `@include`. maybe some keywords are `@conditional`. I trust you in that regard

**Review by theHamsta — APPROVED** [review]
Looks good! Thank you for your contribution

File: queries/erlang/highlights.scm [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  "="
+  "->"
+  "=>"
+ ] @operator
+
+(string) @string
+(variable) @variable
+
+(module_name
+ (atom) @attribute
```
[the remark this answers is not in the document]
Comment (Decodetalkers) [reply queries/erlang/highlights.scm:30]: > 

module name I think..

[the remark this answers is not in the document]
Comment (Decodetalkers) [reply queries/erlang/highlights.scm:30]: image

**Comment by Decodetalkers** [comment]
And add the support for folds by the way

File: queries/erlang/highlights.scm (hunk 3 above)
[the remark this answers is not in the document]
Comment (theHamsta) [reply queries/erlang/highlights.scm:31]: why `@attribute` then?  `@attribute` are used at the moment for rather meta-code annotation like C++ attributes.

[the remark this answers is not in the document]
Comment (Decodetalkers) [reply queries/erlang/highlights.scm]: > why `@attribute` then? `@attribute` are used at the moment for rather meta-code annotation like C++ attributes.

Now I use namespace instead

**Review by kyazdani42 — APPROVED** [review]

**Comment by kyazdani42** [comment]
thanks for the contribution :)

**Pull request merged**